lib/wallaroo_labs/query/query_json.pony

```diff
@@ -121,6 +121,76 @@ primitive PartitionQueryEncoder
     end
     _JsonEncoder(consume entries, _JsonMap)
 
+primitive StateEntityQueryEncoder
```

Comment:
same kind of renaming:
entity -> partitions
partition -> partitions

Reply:
see #2268